content/how-to/bpmn/trigger-workflow.md

```diff
@@ -0,0 +1,144 @@
+
+The synchronous call does not execute further operations until it receives a response from the Rhize server.
+On the other hand, the asynchronous operation frees up the call system to do more work, no matter whether the workflow runs correctly.
+
+The call syntax for the each operation is almost identical.
+For example, compare the syntax for these calls:
+
+{{% tabs %}}
+{{% tab "Synchronous" %}}
+```gql
+mutation SynchronousCall($createAndRunBpmnSyncId: String!, $version: String ) {
+  createAndRunBpmnSync(id: $createAndRunBpmnSyncId, version: $version) {
+    id
+    jobState
+    customResponse
+  }
+}
+```
+{{% /tab %}}
+
+{{% tab "Async" %}}
+```gql
+mutation AsyncCall{
+  createAndRunBPMNSync{
+    id
+    jobState
+    customResponse
+  }
+}
+```
+{{% /tab %}}
+{{% /tabs %}}
+
+The responses for these calls have two differences:
+- For synchronous calls, the returned `JobState` should be a finished value (such as `COMPLETED` or `ABORTED`). Asynchronous calls likely return an in-progress status, such as `RUNNING`.
+- Only the synchronous call receives data in the `customResponse`. For details, refer to the next section.
+
+### `CustomResponse`
+
+The `customResponse` is a special variable to return data in the response to clients that run `createAndRunBPMNSync` operations.
+
+You can set the `customResponse` in any [element]({{< relref "/how-to/bpmn/bpmn-elements" >}}) that has an `Output` or `Input response` parameter.
+It can use any data from the {{< abbr "process variable context" >}}, including variables added on the fly.
+
+Functionally, only the last value of the `customResponse` is returned to the client that sent the response.
+However, you can use conditional branches and different end nodes to add error handling.
+
+For example, this workflow returns `Workflow ran correctly` if the call variables include the message `CORRECT` and an error message in all other cases.
+
+<!-- vale off -->
+{{< bigFigure
+src="/images/bpmn/screenshot-rhize-bpmn-error-handling-custom-response.png"
+alt="A BPMN workflow with customResponse in the output of the end node"
+caption="Download this workflow from [BPMN templates](https://github.com/libremfg/bpmn-templates/)"
+width="80%"
+>}}
+<!-- vale on -->
+
+### Additional properties for workflow calls {#variables-versions}
+
+API operations can also include parameters to add variables to the {{< abbr "process variable context" >}} and to specify a workflow version.
+
+To add variables, use the `variables` input argument.
+Note that the variables are accessible from their root object.
+For example, a workflow would access the following string value at `$.input.message`:
+
+```gql{
+"variables": "{\"input\":{\"message\":\"CORRECT\"}}"
+}
+```
+
+To specify a version, use the `version` property. For example, this input specifies that Rhize run version `3` of the `API_demoCallToRemoteAPI` workflow:
+
+```json
+{
+  "createAndRunBpmnSyncId": "API_demoCallToRemoteAPI",
+  "version": "3"
+}
+```
+
```

Comment:
```suggestion

```

If the `version` property is empty, Rhize runs the active version of the workflow (if one exists).